src/modules/files.ts

```diff
@@ -37,7 +37,7 @@ export default class Files extends API {
    * @param limit List page size (default 5)
    * @returns An array of Thread objects
    */
-  async list(thread?: string, offset?: string, limit?: number) {
+  async list(offset?: string, limit?: number, thread?: string) {
```

Comment:
I get _why_ we want to change this. But this means we'll have a lot of `files.list(undefined, undefined, undefined, thread)`, which isn't great, which is why I had thread in there first. Thoughts?

Reply:
well... maybe i'm being overly hopeful we can match apis across JS, RN and beyond. the node api being exported from go puts thread last, but maybe that is where we should change it. 

https://github.com/textileio/go-textile/blob/master/mobile/files.go#L347

Reply:
Whatever the final answer is, I'd like it to be consistent in all the libs including Go.